metrics/lib/features/dashboard/presentation/pages/dashboard_page.dart

```diff
@@ -1,49 +1,89 @@
+import 'package:fcharts/fcharts.dart';
 import 'package:flutter/material.dart';
 import 'package:metrics/features/dashboard/presentation/state/project_metrics_store.dart';
+import 'package:metrics/features/dashboard/presentation/widgets/circle_percentage.dart';
+import 'package:metrics/features/dashboard/presentation/widgets/sparkline_graph.dart';
 import 'package:states_rebuilder/states_rebuilder.dart';
 
-import '../widgets/circle_percentage.dart';
-
 class DashboardPage extends StatelessWidget {
+  static const double _gradientColorOpacity = 0.4;
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       body: SafeArea(
         child: StateBuilder<ProjectMetricsStore>(
           models: [Injector.getAsReactive<ProjectMetricsStore>()],
           builder: (_, coverageStore) {
```

Comment:
I think we missed this before:

```suggestion
          builder: (_, projectMetricsStore) {
```